remote_cypress_manifest.json

```diff
@@ -0,0 +1,31 @@
+{
+  "schema-version": "1.0",
+  "build": {
+    "name": "OpenSearch Dashboards Functional Test",
+    "version": "2.12.0"
+  },
+  "ci": {
+    "image": {
+    }
+  },
+  "components": [
+    {
+      "name": "OpenSearch-Dashboards",
+      "repository": "opensearch-project/OpenSearch-Dashboards",
+      "workflow-name": "",
+      "ref": "",
+      "opensearch": "test",
+      "opensearch-dashboards": "test",
```

Comment:
As this is an open PR, can we modify these dummy configs to valid ones?

Reply:
Yes for main I have added linux-x64 artifact for now. Release team can choose to update component manifest json for other artifact distribution as needed.